src/Marten.TestHelpers/Marten.TestHelpers.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
**SUGGESTION:** I'd just call this NuGet `Marten.Testing`

Reply:
Will do